chains/evm/deployment/v1_7_0/changesets/configure_chain_for_lanes.go

```diff
@@ -0,0 +1,156 @@
+		for chainSel := range in.RemoteChains {
+			remoteChains = append(remoteChains, chainSel)
+		}
+		return fmt.Sprintf("Configure remote chain connnections on %s: %v", dep, remoteChains)
+	},
+	ResolveInput: func(e cldf_deployment.Environment, cfg ConfigureChainForLanesCfg) (sequences.ConfigureChainForLanesInput, error) {
+		staticAddrs, err := datastore_utils.FindAndFormatEachRef(e.DataStore, []datastore.AddressRef{
+			{
+				ChainSelector: cfg.ChainSel,
+				Type:          datastore.ContractType(router.ContractType),
+				Version:       semver.MustParse("1.2.0"),
+			},
+			{
+				ChainSelector: cfg.ChainSel,
+				Type:          datastore.ContractType(ccv_proxy.ContractType),
+				Version:       semver.MustParse("1.7.0"),
+			},
+			{
+				ChainSelector: cfg.ChainSel,
+				Type:          datastore.ContractType(commit_onramp.ContractType),
+				Version:       semver.MustParse("1.7.0"),
+			},
+			{
+				ChainSelector: cfg.ChainSel,
+				Type:          datastore.ContractType(fee_quoter_v2.ContractType),
+				Version:       semver.MustParse("1.7.0"),
+			},
+			{
+				ChainSelector: cfg.ChainSel,
+				Type:          datastore.ContractType(ccv_aggregator.ContractType),
+				Version:       semver.MustParse("1.7.0"),
+			},
+		}, datastore_utils.ToEVMAddress)
+		if err != nil {
+			return sequences.ConfigureChainForLanesInput{}, fmt.Errorf("failed to resolve contract refs: %w", err)
+		}
+		routerAddr := staticAddrs[0]
+		ccvProxyAddr := staticAddrs[1]
+		commitOnRampAddr := staticAddrs[2]
+		feeQuoterAddr := staticAddrs[3]
+		ccvAggregatorAddr := staticAddrs[4]
+
+		remoteChains := make(map[uint64]sequences.RemoteChainConfig, len(cfg.RemoteChains))
+		for remoteChainSel, remoteConfig := range cfg.RemoteChains {
+			refs := []datastore.AddressRef{remoteConfig.DefaultExecutor}
+			refs = append(refs, remoteConfig.DefaultCCVOffRamps...)
+			refs = append(refs, remoteConfig.LaneMandatedCCVOffRamps...)
+			refs = append(refs, remoteConfig.DefaultCCVOnRamps...)
+			refs = append(refs, remoteConfig.LaneMandatedCCVOnRamps...)
+
+			// Bookmarking for creating the final struct
+			defaultCCVOffRampsEnd := 1 + len(remoteConfig.DefaultCCVOffRamps)
+			laneMandatedCCVOffRampsEnd := defaultCCVOffRampsEnd + len(remoteConfig.LaneMandatedCCVOffRamps)
+			defaultCCVOnRampsEnd := laneMandatedCCVOffRampsEnd + len(remoteConfig.DefaultCCVOnRamps)
+			laneMandatedCCVOnRampsEnd := defaultCCVOnRampsEnd + len(remoteConfig.LaneMandatedCCVOnRamps)
+
+			// Set the chain selector of every ref stored thus far to the chain that we are configuring
+			for i := range refs {
+				refs[i].ChainSelector = cfg.ChainSel
+			}
+
+			addrs, err := datastore_utils.FindAndFormatEachRef(e.DataStore, refs, datastore_utils.ToEVMAddress)
+			if err != nil {
+				return sequences.ConfigureChainForLanesInput{}, fmt.Errorf("failed to resolve contract refs: %w", err)
+			}
+
+			// Resolve CCIPMessageSource separately since it is on the remote chain
```

Comment:
This PR will resolve this issue, we can update the code once merged: https://github.com/smartcontractkit/chainlink-deployments-framework/pull/376